docs/challenge.md

```diff
@@ -3,7 +3,7 @@
 ## 🏠🤖 Challenge  🚀
 The objective of the HomeRobot: OVMM Challenge is to create a platform that enables researchers to develop agents that can navigate unfamiliar environments, manipulate novel objects, and move away from closed object classes towards open-vocabulary natural language. This challenge aims to facilitate cross-cutting research in embodied AI using recent advances in machine learning, computer vision, natural language, and robotics.
 
-Check out the [Neurips 2023 HomeRobot Open-Vocabulary Mobile Manipulation Challenge!](https://aihabitat.org/challenge/2023_homerobot_ovmm/)
+Check out the [CVPR 2024 HomeRobot Open-Vocabulary Mobile Manipulation Challenge!](https://aihabitat.org/challenge/2023_homerobot_ovmm/)
```

Comment:
Is this link correct? It says "NeurIPS 2023 HomeRobot: Open Vocabulary Mobile Manipulation (OVMM) Challenge"

Reply:
I updated it to eval ai page in the last commit.